clang build error on travis when initializing a std::map with {}

We're getting this error with the Mac OS clang build on Travis.

~~~
/Users/travis/build/numenta/nupic.core/src/nupic/algorithms/Connections.cpp:379:24: error: chosen constructor is explicit in copy-initialization
  Activity activity = {{},
~~~

https://travis-ci.org/numenta/nupic.core/jobs/121584947

I don't repro the issue locally on my Mac.

--- a/src/nupic/algorithms/Connections.cpp
+++ b/src/nupic/algorithms/Connections.cpp
@@ -376,7 +376,7 @@ Activity Connections::computeActivity(const vector<Cell>& input,
                                       SynapseIdx synapseThreshold,
                                       bool recordIteration)
 {
-  Activity activity = {{},
+  Activity activity = {map< Cell, std::vector<Segment> >(),
                        vector<UInt32>(nextFlatIdx_, 0)};
 
   for (const Cell& cell : input)

